lib/editor/tunnel.js

```diff
@@ -139,7 +141,7 @@ class EditorTunnel {
                         method: request.method,
                         body: request.body,
                         throwErrors: false,
-                        rejectUnauthorized: false
+                        [thisTunnel.localProtocol]: { rejectUnauthorized: false }
```

Comment:
Reading the `got` docs, this option should only be set for `https`. There is no top-level `http` options object.

Reply:
Hmmm, odd. This option, at top level, causes the ```DeprecationWarning: Got: "options.rejectUnauthorized" ``` warning.

Let me double check, I may have incorrectly applied to both the WS and GOT.

Reply:
reverted in [9d9b421](https://github.com/flowforge/flowforge-device-agent/pull/128/commits/9d9b4210b9a6176ded446883ecf34173a2a95ef5)